modules/combined/doc/content/modules/combined/examples/stm_thermomechanics.md

```diff
@@ -0,0 +1,201 @@
+1. Temperature at center of outer surface -- $T_{2,c}$
+1. Temperature at end of inner surface -- $T_{1,e}$
+1. Temperature at end of outer surface -- $T_{2,e}$
+1. x-displacement at center of inner surface -- $\delta_{x,1,c}$
+1. x-displacement at center of outer surface -- $\delta_{x,2,c}$
+1. x-displacement at end of inner surface -- $\delta_{x,1,e}$
+1. x-displacement at end of outer surface -- $\delta_{x,2,e}$
+1. z-displacement at end of inner surface -- $\delta_{z,1}$
+1. z-displacement at end of outer surface -- $\delta_{z,2}$
+
+
+[fig:qoi] shows geometrically where these QoI's are located in the model.
+
+!media combined/qoi.png caption=Geometric description of quantities of interest id=fig:qoi style=width:75%
+
+## Results
+
+In this exercise, we will use the [statistics](Statistics.md) and [Sobol sensitivity](PolynomialChaosSobolStatistics.md) capabilities available in the stochastic tools module. Two methods are used to perform this analysis. First is using the sampler system to perturb the uncertain properties and retrieve the QoIs which will undergo the analysis. The second is training a [polynomial chaos surrogate](PolynomialChaos.md) and using that reduced order model to sample and perform the analysis.
+
+Using [latin hypercube sampling](LatinHypercubeSampler.md), the thermomechanics model was run with a total of 100,000 samples, the input file is shown by [list:lhs]. A order four polynomial chaos surrogate was training using a Smolyak sparse quadrature for a total of 7,344 runs of the full model. [tab:rt] shows the run-time for sampling the full order model and training and evaluating the surrogate. The training input is shown by [list:train] and the evaluation input is shwon by [list:eval].
+
+!table caption=Stochastic run-time results id=tab:rt
+| Simulation | Samples | CPU Time |
+| :- | :- | :- |
+| Full-Order Sampling | 100,000 | 176 hr  |
+| Polynomial Chaos -- Training | 7,344 | 13.7 hr  |
+| Polynomial Chaos -- Evaluation | 100,000 | 6.8 s  |
+
+!listing combined/examples/stochastic/lhs_uniform.i id=list:lhs caption=Latin hypercube sampling and statistics input file
+
+!listing combined/examples/stochastic/poly_chaos_train_uniform.i id=list:train caption=Polynomial chaos training input file
+
+!listing combined/examples/stochastic/poly_chaos_uniform.i id=list:eval caption=Polynomial chaos evaluation input file
+
+### Statistics
+
+[tab:stat] shows the statistical results of sampling the thermomechanis model and the polynomial chaos surrogate. $\mu$ and $\sigma$ represent the mean and standard deviation of the QoI, and CI is the confidence interval. Note that the confidence interval for the polynomial chaos statistics is not relevant since these values were found analytically using integration techniques. [fig:hist_temp]-[fig:hist_dispz] compares several of the probability distributions of the QoIs between sampling the full-order model and the polynomial chaos surrogate.
```

Comment:
```suggestion
[tab:stat] shows the statistical results of sampling the thermomechanis model and the polynomial chaos surrogate. $\mu$ and $\sigma$ represent the mean and standard deviation of the QoI, and CI is the confidence interval. Note that the confidence interval for the polynomial chaos statistics is not relevant since these values were found analytically using integration techniques. [fig:hist_temp] to [fig:hist_dispz] compares several of the probability distributions of the QoIs between sampling the full-order model and the polynomial chaos surrogate.
```